src/components/menu/menu-item/menu-item.style.js

```diff
@@ -231,12 +236,27 @@ const StyledMenuItemWrapper = styled.a`
         width: 100vw;
         box-sizing: border-box;
       }
+
+      a:focus,
+      button:focus {
+        box-shadow: 0 0 0 var(--borderWidth300) var(--colorsSemanticFocus500);
+        z-index: 1;
+        position: relative;
+      }
+
+      a:focus,
+      a:hover,
+      button:focus,
+      button:hover {
+        background-color: var(--colorsComponentsMenuAutumnStandard600);
+        color: var(--colorsComponentsMenuYang100);
+
+        [data-component="icon"] {
+          color: var(--colorsComponentsMenuYang100);
+        }
+      }
```

Comment:
Similarly to `Submenu`, I would add some additional tests to verify the token colours are being rendered correctly (in addition to the existing ones for the background colour).